tools/nl/eval/README.md

```diff
@@ -9,3 +9,41 @@ python3 fetch_explore_queries.py
 
 This produces a csv file that can be imported to Google Sheet for further manual
 curation to produce golden data for eval purpose.
+
+## Prepare Model and Embedding Endpoints
+
+To evaluate a model and embeddings generated from the model, it needs to be
+deployed to Vertex AI. Concrete steps can be found in
+[model server](../../../model_server/README.md).
+
+Once a model is deployed, generate embeddings and index them in Vertex AI vector
+search. Concreate steps can be found in [build
+embeddings](../embeddings/README.md).
+
+Once model and embeddings endpoints are avaiable, record the endpoints id in
+`vertex_ai_endpoints.yaml` file.
+
+## Eval Unit
+
+One evaluation unit locates in a sub folder that contains a `golden.json` file
+that is a mapping between golden query and ranked stat vars to be matched from
+the vector store. Note there is logic to get to ranked stat vars from ranked
+embedding vectors. Right now it's simply based on the stat vars for a
+description embedding. This can be changed in the future. Multiple evaluation
+unit could exist for different eval sets (like UNSDG, WHO).
```

Comment:
how do you specify multiple evaluation units because looking at the flags, -f only supports a single string? Did you want -f to be a list?

Reply:
Re-worked the paragraph so there is no "eval unit" concept right now. For one, it needs a folder with a eval json, so -f would use only one folder name.